netsupportpolicy.md

```diff
@@ -0,0 +1,16 @@
+.NET support rules:
+1. ODL does not support .NET versions that are out-of-support
+2. .NET support has been tested prior to publishing a version with that .NET support
+3. removing support for a .NET version is a breaking change (and therefore requires a major version increment)
+4. adding support for a .NET version is not a breaking change
+5. ODL major versions are support for at least 1 year after release (TODO this claim was made in the last meeting, but i can't find in the existing doc)
+
+open items:
+1. How do we communicate the lifecycle moves for each ODL major version? (from John)
+    1. My proposal is to let the nuget release notes do this for us.
+2. How do we want to handle testing of .NET release candidates? (from many folks on the team)
+    1. My proposal is to test .NET support after the official .NET release, giving a 3 month runway.

```

Comment:
I think that's too late. I think we have to test with the beta versions of .NET and should have at least a beta version that is verified to work with the .NET release when the .NET release ships, if not sim-ship a GA version of ODL. I'm pretty sure that's what EF does.

Reply:
For your information, below is the release cycle for .NET 8:

![image](https://github.com/user-attachments/assets/666ae7c6-6105-4d99-8af3-b612885c3ce1)

Reply:
we will be part of a beta program instead of being addressed post release; i need to move these dates on the visio file